jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenLayerConfigurations.java

```diff
@@ -46,6 +47,25 @@
    */
   static JavaLayerConfigurations getForProject(
       MavenProject project, Path extraDirectory, AbsoluteUnixPath appRoot) throws IOException {
+    // Since 'jar' is the default packaging type it is not required, so it can be null
```

Comment:
It's OK to do `"war".equals(project.getPackaging())` here and remove the null check and the comment.

Reply:
Done